app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -85,6 +85,30 @@ def disconnect_ems(e = nil)
     super
   end
 
+  def validate_associate_floating_ip
```

Comment:
you can remove those `validate_` methods completely. This reasoning behind the SupportsFeatureMixin is to get rid of those in the codebase :smile: